IRremoteESP8266.cpp

```diff
@@ -33,85 +33,99 @@
 // These versions of MATCH, MATCH_MARK, and MATCH_SPACE are only for debugging.
 // To use them, set DEBUG in IRremoteInt.h
 // Normally macros are used for efficiency
+
+bool  MATCH_MARK_NOLOG(int measured_ticks, int desired) {
+  return  measured_ticks >= TICKS_LOW(desired) && measured_ticks <= TICKS_HIGH(desired+MARK_EXCESS);
+}
+
+bool  MATCH_SPACE_NOLOG(int measured_ticks, int desired) {
+  return   measured_ticks >= TICKS_LOW(desired-MARK_EXCESS) && measured_ticks <= TICKS_HIGH(desired);
+}
+
+#define DEBUG true 
```

Comment:
HI , I wast trying to make the receiver works so whem debuging the code I was seing so much output that not allows understand what matches and watch not matches , it was so useful to allow undertand and them make new fixes , on data the logic is different as it can be one or another , them I moved the logic for not matches on data